app/HutchManager/Models/AgentResultsModel.cs

```diff
@@ -0,0 +1,20 @@
+namespace HutchManager.Models;
+
+public class AgentResultsModel
+{
+  public int Id { get; set; }
+
+  public string Name { get; set; }
+  
+  public string ClientId { get; set; }
+  
+  public List<Data.Entities.DataSource> DataSources { get; set; }
```

Comment:
Don't put entities in here - create a model instead if necessary; or in this case I think just `List<string>` will work?